specification/resources/resource-manager/Microsoft.Resources/preview/2022-08-01-preview/deploymentStacks.json

```diff
@@ -0,0 +1,1353 @@
+{
```

Comment:
Observed that there has been discussion about exception/suppression for other LintDiff issues. What about "UniqueXmsExample" issues?